src/test/java/se/sundsvall/customer/api/model/CustomerDetailsRequestTest.java

```diff
@@ -42,15 +44,21 @@ void hasValidBuilderMethods() {
 			.withPartyId(List.of(partyId))
 			.withCustomerEngagementOrgId(customerEngagementOrgId)
 			.withFromDateTime(fromDateTime);
+		request.setSortBy(List.of("sort1", "sort2"));
 
```

Comment:
Set page and limit as well and verify that they are not the "default values"?

Reply:
Will fix